packages/shared/src/utils/getRemaningCharacterCount.ts

```diff
@@ -0,0 +1,12 @@
+export const MAX_COMMENT_LENGTH = 300;
+export const MAX_POST_LENGTH = 600;
+
+export function getRemaningCharacterCount(message: string, MAX_TEXT_LENGTH: number) {
```

Comment:
typo: Remaining*

Reply:
can we also apply this to `CollectionCreateOrEditForm`? we want the collection descriptions on web to have a new MAX_LENGTH of 2400, and also not count urls.

Reply:
finally, it'd be great to get chatGPT to write some tests for this :)

Reply:
Updated the typo and added it to `CollectionCreateOrEditForm` too


https://github.com/gallery-so/gallery/assets/4480258/f6b4f737-c53c-417c-8989-5ac2d99d1514

Reply:
Added test too!